models/DBRelationalData/area_type.go

```diff
@@ -2,68 +2,53 @@ package DBRelationalData
 
 // var representing test area_type data
 var AreaTypeData = map[string]map[string]interface{}{
-	"England": {
-		"creation_order": 0,
-		"columns":        "name",
-		"values":         "England",
-	},
-	"Wales": {
-		"creation_order": 1,
-		"columns":        "name",
-		"values":         "Wales",
-	},
-	"Stagsden": {
-		"creation_order": 2,
-		"columns":        "name",
-		"values":         "test area 3",
-	},
 	"Country": {
-		"creation_order": 3,
+		"creation_order": 0,
 		"columns":        "name",
 		"values":         "Country",
 	},
 	"Region": {
-		"creation_order": 4,
+		"creation_order": 1,
 		"columns":        "name",
 		"values":         "Region",
 	},
 	"Unitary Authorities": {
-		"creation_order": 5,
+		"creation_order": 2,
 		"columns":        "name",
 		"values":         "Unitary Authorities",
 	},
 	"Combined Authorities": {
-		"creation_order": 6,
+		"creation_order": 3,
 		"columns":        "name",
 		"values":         "Combined Authorities",
 	},
 	"Metropolitan Counties": {
-		"creation_order": 7,
+		"creation_order": 4,
 		"columns":        "name",
 		"values":         "Metropolitan Counties",
 	},
 	"Counties": {
-		"creation_order": 8,
+		"creation_order": 5,
 		"columns":        "name",
 		"values":         "Counties",
 	},
 	"London Boroughs": {
-		"creation_order": 9,
+		"creation_order": 6,
 		"columns":        "name",
 		"values":         "London Boroughs",
 	},
 	"Metropolitan Districts": {
-		"creation_order": 10,
+		"creation_order": 7,
 		"columns":        "name",
 		"values":         "Metropolitan Districts",
 	},
 	"Non-metropolitan Districts": {
-		"creation_order": 11,
+		"creation_order": 8,
 		"columns":        "name",
 		"values":         "Non-metropolitan Districts",
 	},
 	"Electoral Wards": {
-		"creation_order": 12,
+		"creation_order": 9,
```

Comment:
These look fine @joegasewicz - but just checking - does it affect the FK constraint on the `dp-areas-api.area` table (i.e. generate a violation at runtime)?